LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/NetworkServerConfiguration.cs

```diff
@@ -102,7 +102,7 @@ public class NetworkServerConfiguration
         /// <summary>
         /// Gets or sets the gateway netword id.
         /// </summary>
-        public uint NetId { get; set; } = 1;
+        public NetId NetId { get; set; } = new NetId(1);
```

Comment:
I have doubts whether this is meant to be a `NetId`.  I think this might be more the **NwkID** from the specification, the 7 LSB of a **NetID**.

Reply:
Okay, it seems that this is indeed the **NetID** from the specification and therefore using our `NetId` type is the right call here. Sorry, but the confusion was on my end (thus my doubt) that stemmed from not being familiar with the specification in its entirety. Reading the relevant passages and other documents such as the [NetID FAQ](https://lora-alliance.org/wp-content/uploads/2021/01/20200714_NetID_FAQ.pdf) cleared up some of that confusion.